app/src/main/java/ch/epfl/favo/view/tabs/addFavor/FavorPublishedView.java

```diff
@@ -36,15 +35,14 @@
 
 import ch.epfl.favo.MainActivity;
 import ch.epfl.favo.R;
-import ch.epfl.favo.exception.IllegalAcceptException;
-import ch.epfl.favo.exception.IllegalRequestException;
 import ch.epfl.favo.favor.Favor;
 import ch.epfl.favo.favor.FavorStatus;
 import ch.epfl.favo.user.User;
 import ch.epfl.favo.util.CommonTools;
 import ch.epfl.favo.util.DependencyFactory;
 import ch.epfl.favo.view.NonClickableToolbar;
 import ch.epfl.favo.viewmodel.IFavorViewModel;
+import de.hdodenhof.circleimageview.CircleImageView;
```

Comment:
Instead of using an external library, you can achieve the same with a cardview and an imageview. You can look at the UserAccountPage layout for an example but this is also fine, just a suggestion

Reply:
just because I messed things up once with cardview, so just for simplicity I went for external library